### DISCUSSION TITLE
Is there a way to have the executed update commands added to a pull request message?

### DISCUSSION THREAD

**Discussion by bartspiering** [renovatebot/renovate#24062] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Unknown to me, but if needed I can ask

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi!

I think the title already says it all :-) In order to reproduce updates locally in cases of odd looking situations, it would be very nice to have the command(s) used for the update in the corresponding pull request. Is there a way to realize this at the moment?

### What you tried so far.

I googled for it and browsed the docs, but found nothing close enough to what I am looking for

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
No, but you can see the full commands executed within the debug logs
